Assets/HoloToolkit/Utilities/Shaders/HoloToolkitCommon.cginc

```diff
@@ -7,7 +9,7 @@ float4 _NearPlaneFadeDistance;
 // Could instead be done non-linear in projected space for speed
 inline float ComputeNearPlaneFadeLinear(float4 vertex)
 {
-    float distToCamera = -(mul(UNITY_MATRIX_MV, vertex).z);
+    float distToCamera = -(UnityObjectToViewPos(vertex).z);
```

Comment:
This is the only place where `UnityObjectToViewPos` is called (instead of `UnityObjectToClipPos`). This helper method returns a float3 instead of the float4 returned by `mul(UNITY_MATRIX_MV, vertex)`. Since we are only using the z-value, the semantics of this code should be identical to the original code.